### DISCUSSION TITLE
マイページの記事一覧にviews数を表示

### DISCUSSION THREAD

**Discussion by kokosabu** [increments/qiita-discussions#22] [要望・アイデア・不具合]
マイページの記事一覧を開いた時に、LGTM数は表示されているのですが、views数は表示されておらず、知りたいと思った時は記事を開く必要があります。LGTM数同様、viewsも表示していただけると嬉しいです。

どのくらい読まれているか、どんな記事が読まれているかは書く上で知りたい情報だと思います。いかがでしょうか。

**Comment by kyntk** [comment]
@kokosabu
ご意見ありがとうございます。
少し詳しく伺いたいのですが、記事を書く上で｢どんな記事が読まれているか｣を知るためにLGTM数ではなくview数を知りたいモチベーションについて、もう少し教えていただけますでしょうか?

**Reply by kokosabu** [reply]
そこまでLGTMがついたことが無いので、view数をみて読まれているを判断している感じです。
LGTMついていないなりに読んでくださっている人がいるんだなというのがわかります。
( それなりに読まれているのにLGTMつかないような記事なんだなともいえます。 )

本気でやるならGoogleアナリティクスを使うのが正解かもしれないなとも思っています。

**Comment by daikisuyama** [comment]
自分も@kokosabuさんと同じ意見です。
LGTM数とview数の2点から自分の記事を評価しているため、どちらも表示していただけるとありがたいです。
具体的には、view数/LGTM数が低いほど記事の情報の質が高いという傾向があり、これを元に評価しています。
Googleアナリティクスの場合、LGTM数がおそらく表示されないので、この評価が機能しません。

**Comment by kskwtnk** [comment]
@kokosabu @daikisuyama 

お二人ともありがとうございます！

「マイページにおけるview数表示」は、申し訳ないのですが直近では実装しないと思われます
view数だから表示をしたくないわけではなく、Qiitaの「マイページ」という存在は実質的には「ユーザーについての情報がまとまったページ」であり、アクセスしたページと自分のアカウントが一致していたときだけ少し見た目が変わる……というニュアンスです

そして、view数に限らず「マイページに表示して欲しい項目」のリクエストをいただくことはあるのですが、既存のページに「自分のアカウントのときだけ見える情報」を付け足し付け足ししていくとかなりインターフェースとして混沌としてしまいます

もし実施する場合はdev.toなどのように、「自分だけが見えるページ、ダッシュボード的なもの」と「パブリックなプロフィールのページ」を分けることになるだろうと考えているのですが……直近でマイページを改修するのが話に挙がっておらず、なかなか対応できないだろうというのが現状です

ただあくまで「しばらくは対応できなさそう」であり情報設計としてマイページを変えねばとは思っているため、その際にview数表示も検討させていただければと思います！

**Reply by daikisuyama** [reply]
ご回答ありがとうございます。
インターフェスとして混沌としてしまうという点において納得しました。
また、運営側として変更が難しいようであれば、ブラウザの拡張機能等で対応しようと思います。

**Reply by kokosabu** [reply]
承知ました！
